src/panels/config/script/ha-script-editor.ts

```diff
@@ -83,7 +102,78 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
 
   @query("ha-yaml-editor", true) private _editor?: HaYamlEditor;
 
+  private _schema = memoizeOne(
+    (hasID: boolean, useBluePrint?: boolean, currentMode?: string) => {
+      const schema: HaFormSchema[] = [];
```

Comment:
```suggestion
      const schema = [...BASE_GUI_SCHEMA];
```

Reply:
Why even have a BASE_GUI_SCHEMA? You can just define the fields in here. The method is memoized